plugins/omprog/omprog.c

```diff
@@ -489,35 +408,33 @@ writePipe(wrkrInstanceData_t *pWrkrData, uchar *szMsg)
 	len = strlen((char*)szMsg);
 
 	do {
-		checkProgramOutput(pWrkrData);
 		written = write(pWrkrData->fdPipeOut, ((char*)szMsg) + offset, len - offset);
 		if(written == -1) {
 			if(errno == EPIPE) {
 				DBGPRINTF("omprog: program '%s' terminated, will be restarted\n",
```

Comment:
would it be better to expose these messages to the user (via ```LogMsg()```)? Again, nothing new, and if we go that path probably for another PR (so that we can finally merge this one).

Reply:
I agree. And in fact I included this change in another PR I've prepared.